libvmi/driver/xen/xen_events.c

```diff
@@ -246,9 +249,10 @@ status_t process_interrupt_event(vmi_instance_t vmi,
         event->interrupt_event.gla = req->data.regs.x86.rip;
         event->interrupt_event.intr = intr;
         event->interrupt_event.reinject = -1;
-#if VM_EVENT_INTERFACE_VERSION >= 2
-        event->interrupt_event.insn_length = req->u.software_breakpoint.insn_length;
-#endif
+
+        if ( req->version >= 2 )
```

Comment:
This is fine to merge as-is, but I'd like to see a follow-up PR with comments in any place where there's a differing code behavior based upon the vm_event version or the xen maj/minor version #s. 

Previously, the preprocessor statement made the reason for the insn_length statement clear just by its name. Now, the line really can't be taken at face value until an unfamiliar developer learns elsewhere what the version member represents.